storage/badger/cleaner.go

```diff
@@ -82,7 +82,16 @@ func (c *Cleaner) gcWorkerRoutine(ctx irrecoverable.SignalerContext, ready compo
 // We add 20% jitter into the interval, so that we don't risk nodes syncing their GC calls over time.
 // Therefore GC is run every X seconds, where X is uniformly sampled from [interval, interval*1.2]
 func (c *Cleaner) nextWaitDuration() time.Duration {
-	return time.Duration(c.interval.Nanoseconds() + rand.Int63n(c.interval.Nanoseconds()/5))
+	jitter, err := rand.Uint64n(uint64(c.interval.Nanoseconds() / 5))
+	if err != nil {
+		// if randomness fails, do not use a jitter for this instance.
+		// TODO: address the error properly and not swallow it.
+		// In this specific case, `utils/rand` only errors if the system randomness fails
+		// which is a symptom of a wider failure. Many other node components would catch such
+		// a failure.
+		jitter = 0
```

Comment:
log warning? jitter doesn't need cryptographic randomness btw.

Reply:
I agree crypto RNG is not needed for jitters! That's not the only case in these PRs where `math/rand` could be still used btw.
I decided to avoid `math/rand` so that the package is not imported at all in production code. This avoids future accidental re-use of the package in other cases that require secure randomness IMO. It would be good to also enforce this rule somehow as I guess `math/rand` will slip back into the repo after some time..